.github/workflows/pipeline.yml

```
@@ -8,13 +8,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6.5]
+        ruby: [2.6.5, 2.7, '3.0', 3.1]
```

we can test against the latest patch version of ruby 2.6
```suggestion
        ruby: [2.6, 2.7, '3.0', 3.1]
```